spec.html

```diff
@@ -130,7 +148,16 @@ <h1>Conformance</h1>
   <p>A conforming implementation of ECMAScript must not redefine any facilities that are not implementation-defined, implementation-approximated, or host-defined.</p>
   <p>A conforming implementation of ECMAScript may choose to implement or not implement <dfn>Normative Optional</dfn> subclauses. If any Normative Optional behaviour is implemented, all of the behaviour in the containing Normative Optional clause must be implemented. A Normative Optional clause is denoted in this specification with the words "Normative Optional" in a coloured box, as shown below.</p>
   <emu-clause id="sec-conformance.normative-optional" example normative-optional>
-    <h1>Example Clause Heading</h1>
+    <h1>Example Normative Optional Clause Heading</h1>
+    <p>Example clause contents.</p>
+  </emu-clause>
+  <p>A conforming implementation of ECMAScript must implement Legacy subclauses, unless they are also marked as Normative Optional. All of the language features and behaviours specified as “Legacy” have one or more undesirable characteristics. However, their continued usage in existing applications prevents their removal from this specification. These features are not considered part of the core ECMAScript language. Programmers should not use or assume the existence of these features and behaviours when writing new ECMAScript code.</p>
```

Comment:
Can we use a `<dfn>` for legacy so we can link to it from elsewhere?

Reply:
I pushed up a commit to do this. Please review.

Reply:
should we dfn Legacy or legacy? (capital-L or not)

it seems like "Legacy" would more strongly convey it's a special category versus just a dictionary word.

Reply:
+1 for def'ing `Legacy`. That also means we can avoid the emu-not-ref's.

Reply:
Done.